addons/project/i18n/project.pot

```diff
@@ -1401,7 +1395,12 @@ msgstr ""
 
 #. module: project
 #: model:ir.model.fields.selection,name:project.selection__project_project__privacy_visibility__followers
-msgid "Invited employees"
+msgid "Invited internal users"
+msgstr ""
+
+#. module: project
+#: model:ir.model.fields.selection,name:project.selection__project_project__privacy_visibility__portal
+msgid "Invited portal users and all internal users"
```

Comment:
Ideally, in order to keep the diff minimal, I would not change the place of this traduction in the file.